src/components/welcome/index.tsx

```diff
@@ -1,57 +1,77 @@
 import React from 'react'
-import { Button, Divider, Grid, Paper, Typography } from '@mui/material'
+import { Button, Grid, Paper, SvgIcon, Typography } from '@mui/material'
 import { useRouter } from 'next/router'
 import { CREATE_SAFE_EVENTS, LOAD_SAFE_EVENTS } from '@/services/analytics/events/createLoadSafe'
 import Track from '../common/Track'
 import { AppRoutes } from '@/config/routes'
+import SafeList from '@/components/sidebar/SafeList'
+import css from './styles.module.css'
+import NewSafeIcon from '@/public/images/welcome/new-safe.svg'
+import LoadSafeIcon from '@/public/images/welcome/load-safe.svg'
+import Footer from '@/components/common/Footer'
 
 const NewSafe = () => {
   const router = useRouter()
 
   return (
-    <>
-      <Typography variant="h1" mb={2}>
-        Welcome to the Safe.
-      </Typography>
-      <Typography variant="h4" mb={2}>
-        Safe is the most trusted platform to manage digital assets.
-        <br />
-        Here is how to get started:
-      </Typography>
-      <Paper sx={{ padding: 3, maxWidth: '800px' }}>
-        <Grid container gap={3}>
-          <Grid item md={6}>
-            <Typography variant="h2" mb={1}>
-              Create Safe
+    <Grid container spacing={3}>
+      <Grid item xs={12} md={4} lg={3.5}>
+        <div className={css.sidebar}>
+          <SafeList closeDrawer={() => {}} />
+        </div>
+      </Grid>
+      <Grid item xs={12} md={8} lg={8.5}>
+        <div className={css.main}>
+          <div className={css.content}>
+            <Typography variant="h1" fontSize={52} lineHeight={1} letterSpacing={-1.5} mb={1}>
+              Welcome to the Safe.
```

Comment:
Typography-wise, should we remove the final dot?
The two blocks below (Create Safe and Load Safe) don't have a dot in the headings.